weka_upgrade_checker/weka_upgrade_checker.py

```diff
@@ -388,25 +388,31 @@ def weka_cluster_checks(skip_mtu_check):
     else:
         GOOD("✅  Weka user login successful")
 
+    def clean_version_string(version_string):
+        """Remove non-numeric suffixes from version string to use with python3.10"""
+        return version_string.split("-")[0]
+
     INFO("WEKA IDENTIFIED")
     weka_info = json.loads(subprocess.check_output(["weka", "status", "-J"]))
     cluster_name = weka_info["name"]
     weka_status = weka_info["status"]
     uuid = weka_info["guid"]
-    weka_version = weka_info["release"]
+    weka_versions = weka_info["release"]
     usable_capacity = weka_info["capacity"]["total_bytes"]
     weka_buckets = weka_info["buckets"]["total"]
 
     GOOD(
-        f"✅  CLUSTER:{cluster_name} STATUS:{weka_status} VERSION:{weka_version} UUID:{uuid}"
+        f"✅  CLUSTER:{cluster_name} STATUS:{weka_status} VERSION:{weka_versions} UUID:{uuid}"
     )
 
-    if any(c.isalpha() for c in weka_version) and not weka_version.endswith("-hcfs"):
+    if any(c.isalpha() for c in weka_versions) and not weka_versions.endswith("-hcfs"):
         INFO("CHECKING WEKA HOTFIX VERSION")
         WARN(
-            f"⚠️  The cluster maybe running a specialized hotfix version of Weka {weka_version}; confirm that required hotfixes are included in upgrade target version."
+            f"⚠️  The cluster maybe running a specialized hotfix version of Weka {weka_versions}; confirm that required hotfixes are included in upgrade target version."
         )
```

Comment:
maybe --> **_may be_**

Reply:
that is not a typo I had to change version to versions for this check to adhere to pep440